File: c#/iterator/Ejm2/Candidate.cs

namespace Iterator.Ejm2
{
    /// <summary>
    /// Representa un candidato
    /// </summary>
    public class Candidate
    {

        private string Name;
        private string CertificationType;
        private string Location;

        public Candidate(string name, string certificationType, string location)
        {
            this.Name = name;
            this.CertificationType = certificationType;
            this.Location = location;
        }

        public string GetName()
        {
            return Name;
        }

        public void SetName(string name)
        {
            this.Name = name;
        }

        public string GetCertificationType()
        {
            return CertificationType;
        }

        public void SetCertificationType(string certificationType)
        {
            this.CertificationType = certificationType;
        }

        public string GetLocation()
        {
            return Location;
        }

        public void setLocation(string location)
        {
            this.Location = location;
        }
    }

}

File: java/iterator/ej2/Candidate.java
package patrones.iterator.ej2;

public class Candidate {

	private String name;
	private String certificationType;
	private String location;

	public Candidate(String name, String certificationType, String location) {
		super();
		this.name = name;
		this.certificationType = certificationType;
		this.location = location;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getCertificationType() {
		return certificationType;
	}

	public void setCertificationType(String certificationType) {
		this.certificationType = certificationType;
	}

	public String getLocation() {
		return location;
	}

	public void setLocation(String location) {
		this.location = location;
	}
}
